test/performance-charts/vitest.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vitest/config';
+import codspeedPlugin from '@codspeed/vitest-plugin';
+import react from '@vitejs/plugin-react';
+
+export default defineConfig({
+  plugins: [codspeedPlugin(), react()],
+  test: {
+    environment: 'jsdom',
```

Comment:
Are you sure that jsdom is where you want to test performance? Imo, as an emulated test environment it has more chances of diverging from what users see in real-world cases.